src/main/java/com/softserve/kickscootertrip/converters/DtoToEntity.java

```diff
@@ -0,0 +1,21 @@
+package com.softserve.kickscootertrip.converters;
+
+
+import com.softserve.kickscootertrip.model.dto.TripDto;
+import com.softserve.kickscootertrip.model.entity.TripEntity;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DtoToEntity implements Converter<TripDto, TripEntity> {
```

Comment:
`DtoToEntity` is a general name meaning that it can convert any DTO to any entity. I think `TripDtoToTripConverter` would be better,